utils/casctl

```diff
@@ -1,10 +1,9 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3.6
```

Comment:
should be 
```#!/usr/bin/env python3```
since I guess someone could have python3.7 and would still want this to work. Comment applies to the other file as well.

Reply:
done